roles/addons/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+# defaults file for addons
+
+# determines if we are on the first master instance.
+is_first_master: "{% if inventory_hostname == groups['masters'][0] %}true{% else %}false{% endif %}"
```

Comment:
cool, we'll create pr for making kubectl available then we can discuss how to merge/integrate